src/pages/components/input.mdx

```diff
@@ -123,6 +123,8 @@ import IconsCode from '!!raw-loader!../../examples/forms/inputs/MediaInput.tsx';
 ### Placeholder
 
 Placeholder text provides the user with example content or additional context.
+Use icon to provide a visual affordance when the [label is hidden](#hidden-label)
+(for example, Search).
```

Comment:
Is `Search` a proper noun in the context of this component?

Reply:
Open to suggestions. The intent here was to show a specific use case, like this one on [Combobox](https://garden.zendesk.com/components/combobox#search).